modules/ROOT/pages/sap/sap-connector-config-topics.adoc

```diff
@@ -0,0 +1,171 @@
+from your SAP site access.
+. Click Test Connection to ensure that you have the correct login information.
+
+== Define Extended Properties
+
+To define extended properties for the SAP connector global element, complete the following steps:
+
+. Navigate to the Advanced tab on the General pane:
++
+image::sap/sap-connector-advanced-tab.png[Lower Advanced Tab]
++
+. Locate the Extended Properties section at the top of the window.
+. Select the Edit inline option.
+. Click the plus button ('+') to add a new extended property.
+
+For this to work you must set the property name, as defined by SAP, in your configuration.
+
+=== SAP Inbound
+
+There are no special Advanced properties for this configuration.
+
+=== SAP Outbound
+
+[%header%autowidth.spread]
+|===
+|Field |XML Attribute |Description |Default Value
+|Display Name |name |The reference name of the endpoint used internally by Mule configuration. |
+|Encoding |encoding |Canonical name of an encoding. See https://docs.oracle.com/javase/7/docs/technotes/guides/intl/encoding.doc.html[Oracle's Supported Encodings]. |
+|Default idocument Version |defaultIdocumentVersion |This version is used when sending the IDoc. Values for the IDoc version correspond to IDOC_VERSION_xxxx constants in com.sap.conn.idoc.IDocFactory. |
+|Disable Function template cache flag |disableFunctionTemplateCacheFlag |Indicates if the function template cache should be disabled. |false
+|Evaluate response flag |evaluateResponseFlag |Indicates if an error response should throw an exception or let the user handle it as another result. |false
+|Log trace flag |logTraceFlag |Indicates if the trace should be logged on the Mule server. |false
+|Use SAP default format |useSAPFormat |Indicates to use SAP default format when working with BAPI functions over sRFC. |false
+|===
+
+== SAP JCo Notes
+
+SAP JCo facilitates communication between an SAP backend system and a Java application. SAP JCo allows Java programs to connect to SAP systems and invoke Remote Function Modules. SAP JCo also allows parsing of IDocs (SAP Intermediate Documents), among other object types. Both inbound and outbound communications are supported.
+
+* Java API: Handles dynamic metadata lookup and caching. It implements JCO.Function, which is the container for parameters and/or tables for the SAP Function Module (BAPI) in Java. Java applications are built on top of the Java API.
+
+* JNI (Java Native Interface): Originally, SAP created libraries in C language to allow direct RFC calls to SAP, to manipulate with data. JCo wraps C libraries in Java to provide platform-native access into the SAP system. RFC Middleware uses RFC Library through (JNI) Layer.
+
+* RFC (Remote Function Call): Communication with the SAP system is performed over the RFC protocol. RFC means calling BAPI or triggering IDoc processing that runs in another system as calling program. The RFC interface enables function calls between two SAP systems or between the SAP and external system.
```

Comment:
RFC means calling BAPI or triggering IDoc processing
  >  A remote function call calls BAPI or triggers IDoc processing

Reply:
that runs in another system as calling program
  >  that runs in another system as a calling program

Reply:
or between the SAP and external system
  >  or between the SAP system and an external system